modules/toolbox/ml_runners.py

```diff
@@ -20,7 +20,9 @@ def run_regressions(validation_pack, package):
         item = getattr(scikit_regression_learners, function)
```

Comment:
Everywhere else is well commented as to whats happening.   Can you maybe add some clarification in the code to describe what is happening here model[0], model[1]. What are they?